version_info.json

```diff
@@ -2,9 +2,8 @@
   "pinned-versions": {
     "iree-compiler": "20220809.230",
     "iree-runtime": "20220809.230",
-    "iree-tools-xla": "20220809.230",
-    "jax": "0.3.15",
+    "iree-tools-xla": "20220707.197",
```

Comment:
This update shouldn't be in here, shoudl it?

Reply:
Was keeping it in originally to test the auto-update. Reverted for this to land with only scripting changes.